src/cargo/core/compiler/context/mod.rs

```diff
@@ -604,7 +604,7 @@ impl<'a, 'cfg> Context<'a, 'cfg> {
     /// Returns whether when `unit` is built whether it should emit metadata as
     /// well because some compilations rely on that.
     pub fn rmeta_required(&self, unit: &Unit) -> bool {
```

Comment:
does this now pull its weight as a separate function?

Reply:
I think this is ok for now in the sense that it keeps the field private to just this module